qemu/tests/balloon_check.py

```diff
@@ -614,5 +616,20 @@ def run(test, params, env):
             return
     try:
         balloon_test.reset_memory()
+        if params.get("balloon_opt_free_page_reporting") == "yes" and \
+                balloon_test.params.get("os_type") == "linux":
+            get_res_cmd = params["get_res_cmd"] % balloon_test.vm.get_pid()
+            memhog_cmd = params["memhog_cmd"]
+            consumed_mem = float(normalize_data_size(params["consumed_mem"]))
+            res1 = float(normalize_data_size(process.getoutput(get_res_cmd)))
+            session = balloon_test.vm.wait_for_login()
+            session.cmd_output_safe(memhog_cmd)
+            res2 = float(normalize_data_size(process.getoutput(get_res_cmd)))
+            time.sleep(30)
```

Comment:
I remember that it's not preferable time.sleep (xx)

Reply:
The step is to check if memory usage returns back after 30s. Do you have better idea? I think it's not necessary to use wait_for() and check res again and again in 30s. What do you think?